src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+    list-style-type: none;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    text-transform: uppercase;
+    height: 60px;
+    line-height: 60px;
+    text-decoration: none;
+    color: #060b35;
+
+    transition: all $transition-duration;
```

Comment:
its bad practice to set transition property ALL, you need to animate just some particular properties